diffmatchpatch/diff_test.go

```diff
@@ -998,8 +998,38 @@ func TestDiffText(t *testing.T) {
 }
 
 func TestDiffDelta(t *testing.T) {
+
+	type TestCase struct {
+		Text               string
+		Delta              string
+		ErrorMessagePrefix string
```

Comment:
Since ErrorMessagePrefix is part of the expected members please add a new line before the field. That is just the style of the other test functions.